cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,314 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/gen/cloudwatch"
+	"github.com/awslabs/aws-sdk-go/gen/ec2"
+	"github.com/awslabs/aws-sdk-go/gen/elb"
+)
+
+const (
+	awsCPU                string = "aws.ec2.cpu"
+	awsNetwork            string = "aws.ec2.net.bytes"
+	awsEC2DiskBytes       string = "aws.ec2.disk.bytes"
+	awsEC2DiskOps         string = "aws.ec2.disk.ops"
+	awsStatusCheckFailed  string = "aws.ec2.status.failed"
+	awsELBLatencyMin      string = "aws.elb.latency.minimum"
+	awsELBLatencyMax      string = "aws.elb.latency.maximum"
+	awsELBLatencyAvg      string = "aws.elb.latency.average"
+	awsELBHostsHealthy    string = "aws.elb.hosts.healthy"
+	awsELBHostsUnHealthy  string = "aws.elb.hosts.unhealthy"
+	awsEC2CPUDesc         string = "The average CPU Utilization, gathered at a 60 second interval and averaged over five minutes."
+	awsEC2NetBytesDesc    string = "The average bytes transmitted or received via network, gathered at a 60 second interval and averaged over five minutes."
+	awsEC2DiskBytesDesc   string = "The average bytes written or read via disk, gathered at a 60 second interval and averaged over five minutes."
+	awsEC2DiskOpsDesc     string = "The average disk operations, either written or read, gathered at a 60 second interval and averaged over five minutes."
+	awsEC2StatusCheckDesc string = "The EC2 Status Check, which includes both instance-level and system-level drill-down, gathered every 60 seconds."
+	awsELBLatencyDesc     string = "The minimum, maximum and average latency as reported by the load balancer, gathered at a 60 second interval and averaged over five minutes."
+	awsELBHostCountDesc   string = "The number of instances in what the Elastic Load Balancer considers a healthy state, gathered every 60 seconds."
+)
+
+func AWS(accessKey, secretKey, region string) {
```

Comment:
This one would need to be caps though since we need it exported to call it in main.go